docker-build-ppc64el.sh

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env bash
+
+#
+# sudo yum install -y docker
+# sudo systemctl start docker.service
+#
+# OLDDIR=$(pwd)
+# git clone https://github.com/ariya/phantomjs.git
+# pushd phantomjs/
+# cp ${OLDDIR}/docker-build-ppc64el.sh deploy/
+# sudo docker run -v $PWD:/src -it --privileged --name phantomjs_ppc64el docker.io/ppc64le/debian:jessie /src/deploy/docker-build-ppc64el.sh
+# sudo docker cp phantomjs_ppc64el:/src/phantomjs ${HOME}/bin/
+# sudo docker rm phantomjs_ppc64el
+# popd
+#
+
+set -e
+
+SOURCE_PATH=/src
+BUILD_PATH=$HOME/build
+
+echo "Installing packages for development tools..." && sleep 1
+apt-get -y update
```

Comment:
That script is to build the required phantomjs library as documented here https://fedoraproject.org/wiki/User:Hamzy/phantomjs_ppc64le

Since I don't know if the phantomjs project will accept it (the master branch has been broken for over a year), I am including it here.